book/7-solution-delivery/deployment-requirements-gathering.md

```diff
@@ -0,0 +1,33 @@
+# What are some questions to be asked to the project sponsor to understand their deployment requirements?
+
+Contributor(s): Siti Nuruljannah Baharudin
+
+## General
+1. Where would the final solution be deployed? 
+    - Options may include a cloud service, an on-premise server, or an edge device. If the specifications are fixed, the client should provide them as soon as possible in order for the team to assess whether they are sufficient for the AI solution to operate on.
+2. How should the application be packaged for deployment? 
+    - Although containerising the application is a common solution, some organisations may have restrictions on the containerisation technology allowed on their servers, or even disallow containerisation altogether. If so, it is recommended that you assess the impact on the deployment effort. For instance, the team may lack expertise in the containerisation technology used, or when containerisation is not used, more system integration testing may be required. These would result in additional effort to prepare the deployment package, which may impact the project schedule.
+3. Are there any restrictions or preferences regarding the operating system on which the solution should be deployed? 
+4. Would a staging environment or test device be provided for developers to test with prior to initial deployment? 
+    - If not provided, you would need to commission an internal environment that closely mirrors the specifications of the environment on which the solution would eventually be deployed.
+
+## Data
+1. Where would the data to be used for retraining and inference be located in the production environment? 
+2. Where would the model outputs be stored? Is there any storage size limit on this location? 
+3. If the data storage is located remotely, what is the authentication method to be used to access it? 
+
+## Usage
```

Comment:
Suggest adding a general comment in this section to say that often, the requirements and considerations for inference are more critical compared to retraining, as the latter is usually performed less frequently and is less time-sensitive.

Reply:
Only 1 comment above, rest looks good Jannah.